spec/recipes/default_spec.rb

```diff
@@ -1,14 +1,30 @@
 require 'spec_helper'
 
-# Write unit tests with ChefSpec - https://github.com/sethvargo/chefspec#readme
 describe 'pantry::default' do
-  let(:chef_run) { ChefSpec::Runner.new.converge(described_recipe) }
+  context 'Mac OS X' do
+    let(:chef_run) do
+      ChefSpec::SoloRunner.new(platform: 'mac_os_x', version: '10.10').
```

Comment:
Please use ZeroRunner. I have no intention of supporting Chef Solo (directly) and would rather testing is done with Zero.